### ISSUE TITLE
hi,i am coming again..this time ,i want to seek advice from another ssd train process..hope for your save

### ISSUE THREAD

**Issue by mathCrazyy** [KleinYuan/tf-object-detection#4]
the link is  https://github.com/balancap/SSD-Tensorflow  , and I follow the introduce to make my tfrecord, for a simple try, I use one class ,and got 13 tfrecords, but , may be there are too much place I shoud to modify ,I do not know how should I do .Beginning ,I did not modify anywhere in train_ssd_network.py,and things go likely, later I modify the num_classes to 1 from 21,things to be bad. and cant go on. The traing_config.txt Is 
`
# =========================================================================== #
# Training | Evaluation flags:
# =========================================================================== #
{'adadelta_rho': 0.95,
 'adagrad_initial_accumulator_value': 0.1,
 'adam_beta1': 0.9,
 'adam_beta2': 0.999,
 'batch_size': 16,
 'checkpoint_exclude_scopes': None,
 'checkpoint_model_scope': None,
 'checkpoint_path': './checkpoints/ssd_300_vgg.ckpt/ssd_300_vgg.ckpt',
 'clone_on_cpu': False,
 'dataset_dir': '/home/wuchunsheng/project/detection_task/data/gesture_1',
 'dataset_name': 'pascalvoc_2012',
 'dataset_split_name': 'train',
 'end_learning_rate': 0.0001,
 'ftrl_initial_accumulator_value': 0.1,
 'ftrl_l1': 0.0,
 'ftrl_l2': 0.0,
 'ftrl_learning_rate_power': -0.5,
 'gpu_memory_fraction': 0.7,
 'ignore_missing_vars': False,
 'label_smoothing': 0.0,
 'labels_offset': 0,
 'learning_rate': 0.001,
 'learning_rate_decay_factor': 0.94,
 'learning_rate_decay_type': 'exponential',
 'log_every_n_steps': 10,
 'loss_alpha': 1.0,
 'match_threshold': 0.5,
 'max_number_of_steps': None,
 'model_name': 'ssd_300_vgg',
 'momentum': 0.9,
 'moving_average_decay': None,
 'negative_ratio': 3.0,
 'num_classes': 21,
 'num_clones': 1,
 'num_epochs_per_decay': 2.0,
 'num_preprocessing_threads': 4,
 'num_readers': 4,
 'opt_epsilon': 1.0,
 'optimizer': 'adam',
 'preprocessing_name': None,
 'rmsprop_decay': 0.9,
 'rmsprop_momentum': 0.9,
 'save_interval_secs': 600,
 'save_summaries_secs': 60,
 'train_dir': '/home/wuchunsheng/project/detection_task/data/gesture_1',
 'train_image_size': None,
 'trainable_scopes': None,
 'weight_decay': 0.0005}

# =========================================================================== #
# SSD net parameters:
# =========================================================================== #
{'anchor_offset': 0.5,
 'anchor_ratios': [[2, 0.5],
                   [2, 0.5, 3, 0.3333333333333333],
                   [2, 0.5, 3, 0.3333333333333333],
                   [2, 0.5, 3, 0.3333333333333333],
                   [2, 0.5],
                   [2, 0.5]],
 'anchor_size_bounds': [0.15, 0.9],
 'anchor_sizes': [(21.0, 45.0),
                  (45.0, 99.0),
                  (99.0, 153.0),
                  (153.0, 207.0),
                  (207.0, 261.0),
                  (261.0, 315.0)],
 'anchor_steps': [8, 16, 32, 64, 100, 300],
 'feat_layers': ['block4', 'block7', 'block8', 'block9', 'block10', 'block11'],
 'feat_shapes': [(38, 38), (19, 19), (10, 10), (5, 5), (3, 3), (1, 1)],
 'img_shape': (300, 300),
 'no_annotation_label': 21,
 'normalizations': [20, -1, -1, -1, -1, -1],
 'num_classes': 21,
 'prior_scaling': [0.1, 0.1, 0.2, 0.2]}

# =========================================================================== #
# Training | Evaluation dataset files:
# =========================================================================== #
['/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_000.tfrecord',
 '/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_001.tfrecord',
 '/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_002.tfrecord',
 '/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_003.tfrecord',
 '/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_004.tfrecord',
 '/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_005.tfrecord',
 '/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_006.tfrecord',
 '/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_007.tfrecord',
 '/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_008.tfrecord',
 '/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_009.tfrecord',
 '/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_010.tfrecord',
 '/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_011.tfrecord',
 '/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_012.tfrecord',
 '/home/wuchunsheng/project/detection_task/data/gesture_1/voc_2012_train_013.tfrecord']

`
can you point me where should I pay more attentions and what shoud I modify ,thank you ,and hope for your help.

**Comment by KleinYuan** [comment]
Hi,
I’m happy to help, even tho this is not quite like an issue on my repo.

Could u close this issue and join this slack channel, find me, and I can help u solve this issue?

https://deep-learning-geeks-slack.herokuapp.com/

**Comment by mathCrazyy** [comment]
ok,i am happy do that, thank you very much.
